pkg/get/get_test.go

```diff
@@ -151,3 +151,66 @@ func Test_DownloadWindows(t *testing.T) {
 		t.Fatalf("want: %s, got: %s", want, got)
 	}
 }
+
+func Test_DownloadHelmDarwin(t *testing.T) {
+	tools := MakeTools()
+	name := "helm"
+	var tool *Tool
+	for _, target := range tools {
+		if name == target.Name {
+			tool = &target
+			break
+		}
+	}
+
+	got, err := tool.GetURL("darwin", arch64bit, tool.Version)
+	if err != nil {
+		t.Fatal(err)
+	}
+	want := "https://get.helm.sh/helm-v3.2.4-darwin-amd64.tar.gz"
+	if got != want {
+		t.Fatalf("want: %s, got: %s", want, got)
+	}
+}
+
+func Test_DownloadHelmLinux(t *testing.T) {
+	tools := MakeTools()
+	name := "helm"
+	var tool *Tool
+	for _, target := range tools {
+		if name == target.Name {
+			tool = &target
+			break
+		}
+	}
+
+	got, err := tool.GetURL("linux", arch64bit, tool.Version)
+	if err != nil {
+		t.Fatal(err)
+	}
+	want := "https://get.helm.sh/helm-v3.2.4-linux-amd64.tar.gz"
+	if got != want {
+		t.Fatalf("want: %s, got: %s", want, got)
+	}
+}
+
+func Test_DownloadHelmWindows(t *testing.T) {
+	tools := MakeTools()
+	name := "helm"
+	var tool *Tool
+	for _, target := range tools {
+		if name == target.Name {
+			tool = &target
+			break
+		}
+	}
+
+	got, err := tool.GetURL("mingw64_nt-10.0-18362", arch64bit, tool.Version)
+	if err != nil {
+		t.Fatal(err)
+	}
+	want := "https://get.helm.sh/helm-v3.2.4-windows-amd64.zip"
```

Comment:
Is the un-tar code really going to work with a zip?

Reply:
My gut feeling is: no. However, there is a Go library that can uncompress zips -> https://golang.org/src/archive/zip/example_test.go

Reply:
I may be wrong here, 

```
tar -xvf helm-v3.2.4-windows-amd64.zip 
x windows-amd64/
x windows-amd64/helm.exe
x windows-amd64/README.md
x windows-amd64/LICENSE
```

Reply:
`bsdtar` can uncompress zip archives, but I would think the Golang tar code cannot. In any case using archive/zip for zips would be better suited.

Reply:
@alexellis you're right. it doesn't work with zip file. I add `Unzip` function to deal with that.

Since i don't have a windows machine, I just hardcode a zip url and test . it's able to extract the zip files in the pic below

![image](https://user-images.githubusercontent.com/627278/85163684-6605a400-b28d-11ea-809e-3787c11079e5.png)